scripts/backend_tests_incomplete_coverage.txt

```diff
@@ -9,7 +9,6 @@ core.domain.suggestion_registry_test
 core.controllers.feedback_test
 core.domain.topic_domain_test
 core.domain.rights_manager_test
-core.domain.story_domain_test
```

Comment:
Why are you removing this? This PR is for maintaining 100% coverage of `core.domain.exp_domain`, right?

Reply:
Yes this PR is for exp_domain. I did not edit this file manually, I'm assuming one of the scripts changed this? Can easily revert this change though